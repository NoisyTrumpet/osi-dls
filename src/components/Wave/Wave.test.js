import React from 'react';
import { shallow } from 'enzyme';
import toJson from 'enzyme-to-json';
import { Wave } from './';

describe('<Wave />', () => {
  test('should render wave component', () => {
    const wrapper = shallow(
      <Wave colorTop="brand-white" colorBottom="gray1" reverseDirection />,
    );
    expect(toJson(wrapper)).toMatchSnapshot();
  });
});
