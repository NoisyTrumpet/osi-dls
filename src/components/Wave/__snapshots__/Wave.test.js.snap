// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Wave /> should render wave component 1`] = `
<div
  className="wave"
>
  <InlineSVG
    className="wave-inlinesvg color-top-brand-white color-bottom-gray1 reverse-direction"
    description="Decorative wave image"
    focus={false}
    path="[object Object]"
    preProcessor={null}
    role="img"
    title="Wave svg"
  />
</div>
`;
